modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,257 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
 ======
 =====
 
+=== Access Neo4j on port `:80` or `:443`
+
+The Neo4j Helm charts provide a Helm chart that allows you to use a Kubernetes Ingress to access Neo4j on port `:80` or `:443`.
+The Helm chart is called _neo4j/neo4j-reverse-proxy_ and is available on the Neo4j Helm repository from version 5.12.0.
+For more information about Kubernetes Ingress, see the link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes official documentation -> Ingress].
+
+The Helm chart creates a reverse proxy that is configured to route traffic to the Neo4j service URL using the `serviceName`, `namespace`, and `domain` values.
+For example, if the `serviceName` is _standalone-admin_, the `namespace` is _default_, and the `domain` is _cluster.local_, then the Neo4j service URL is _standalone-admin.default.svc.cluster.local_. +
+For Neo4j clusters, the Neo4j headless service is used to route the traffic to the cluster instances.
```

Comment:
Neo4j headless service "can" be used ....its not compulsory to use headless service ...users can use loadbalancer service too technically